snuba/admin/google.py

```diff
@@ -0,0 +1,31 @@
+from urllib.parse import urlencode
+
+from googleapiclient.discovery import Resource
+
+
+def check_transitive_membership(service: Resource, parent: str, member: str) -> bool:
+    try:
+        query_params = urlencode({"query": "member_key_id == '{}'".format(member)})
+        request = (
+            service.groups().memberships().checkTransitiveMembership(parent=parent)
+        )
+        request.uri += "&" + query_params
+        response = request.execute()
+        return bool(response["hasMembership"])
+    except Exception as e:
+        print(e)
```

Comment:
This swallows the error and makes sure it goes nowhere. `logging.exception` will at least send it to sentry.

Also a blanket try catch like this is too permissive. This will catch everything like `KeyError` and `AttributeError` which you probably want to fail loudly. 

Combine this with the fact that there are no tests for this function (or `get_group_id`) means that you don't have a reliable authentication mechanism

Reply:
You're right, I'll refactor this to log the error.